agr_literature_service/api/crud/topic_entity_tag_crud.py

```diff
@@ -31,7 +33,7 @@
 ATP_ID_SOURCE_CURATION_TOOLS = "curation_tools"
 
 
-def create_tag(db: Session, topic_entity_tag: TopicEntityTagSchemaPost) -> int:
+def create_tag(db: Session, topic_entity_tag: TopicEntityTagSchemaPost) -> dict:
```

Comment:
oh, how did this work as an int before ?

Reply:
Oh, it used to be an int, nevermind

Reply:
It was returning the newly generated topic_entity_tag_id before so it was int @azurebrd

Reply:
I didn't pay attention to that 'int' earlier until the unittest failed =)

Reply:
Thanks @sweng66 :)